cmd/tpm-v2.c

```diff
@@ -8,11 +8,43 @@
 #include <command.h>
 #include <dm.h>
 #include <log.h>
+#include <malloc.h>
 #include <mapmem.h>
 #include <tpm-common.h>
 #include <tpm-v2.h>
 #include "tpm-user-utils.h"
 
+static int do_tpm2_get_random(struct cmd_tbl *cmdtp, int flag, int argc,
+			      char *const argv[])
+{
+	struct udevice *dev;
+	char *buffer;
+	u32 len;
+	int ret;
+
+	ret = get_tpm(&dev);
+	if (ret) {
+		printf("Cant get tpm\n");
```

Comment:
We might not need this verbose print here.

Reply:
IMO it helps

Reply:
I think this commit is a good candidate for upstream. Good extension of functionality.
IMHO this could make upstreaming this commit a little bit difficult.

Reply:
yes, at the time of submitting this local pull request some months ago I said I wouldn't spend time upstreaming it because it required the tests. if you add the tests then sure, lets do it!